src/prometheus/ast.h

```diff
@@ -0,0 +1,126 @@
+#ifndef AST_H
+#define AST_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef enum {
+  PR_COUNTER,
+  PR_GAUGE,
+  PR_HISTOGRAM,
+  PR_SUMMARY,
+  PR_UNTYPED
+} pr_metric_type_t;
+
+typedef enum {
+  PR_METRIC_ENTRY,
+  PR_COMMENT_ENTRY,
+  PR_TYPE_ENTRY,
+  PR_HELP_ENTRY
+} pr_entry_type_t;
+
+typedef enum { PR_METRIC_FAMILY_ITEM, PR_COMMENT_ITEM } pr_item_type_t;
```

Comment:
Nicer to have all typedefs layed out similarly:
```suggestion
typedef enum {
  PR_METRIC_FAMILY_ITEM,
  PR_COMMENT_ITEM
} pr_item_type_t;
```

Or was this something the format checker insisted on?

Reply:
Unfortunately, clang-format repository check insisted on one-line enum